app/assets/stylesheets/new_styles/_forms.scss

```diff
@@ -103,3 +103,7 @@ form.block-form {
     ::placeholder { text-transform: uppercase; }
   }
 }
+
+.form_errors {
+  color : #b94a48;
```

Comment:
Another solution could be the `text-error` class [Bootstrap](http://getbootstrap.com/2.3.2/base-css.html#typography) has. Just put the error messages in an element with that class and the text color will be red.